src/deploy_tools/models/binary_app.py

```diff
@@ -0,0 +1,43 @@
+from enum import StrEnum
+from typing import Literal
+
+from pydantic import Field
+
+from .parent import ParentModel
+
+
+class HashType(StrEnum):
+    """
+    Type of hash to use for the binary.
+    """
+
+    SHA256 = "sha256"
+    SHA512 = "sha512"
+    MD5 = "md5"
+    NONE = "none"
+
+
+class BinaryApp(ParentModel):
+    """
+    Represents a standalone Binary application.
+    This will fetch a standalone binary, validate its hash and add its
+    location to that path.
+    """
+
+    app_type: Literal["binary"]
+    name: str = Field(
+        ...,
+        description="Binary filename to use locally",
+    )
+    url: str = Field(
+        ...,
+        description="URL to download the binary from.",
+    )
+    hash_type: HashType = Field(
```

Comment:
Okay, this one is very picky, but I would add sha256 as default and move it under has

Reply:
I found that having a default made the yaml less clear.

I don't understand what the second clause was meant to mean?

Reply:
I meant fields in order: app_type, name, url, hash, hash_type.

But it's up to you, especially as it's not going to have a default. I just normally put qualifiers / optionals lower down the list.

Reply:
done